### PR TITLE
Added Accuracy, Misscount, and Max Combo columns

### PR DESCRIPTION
**Pull request by Finadoggie**
more nice qol, this time with some help from StanR

### CODE REVIEW

**Comment by Finadoggie** [comment]
shit, that's not supposed to happen

File: PerformanceCalculator/Profile/ProfileCommand.cs [hunk 1]
```diff
@@ -87,12 +87,32 @@ public override void Execute()
                 var performanceCalculator = ruleset.CreatePerformanceCalculator(working, score.ScoreInfo);
                 Trace.Assert(performanceCalculator != null);
 
+                var perfCalc = ruleset.CreatePerformanceCalculator(working, score.ScoreInfo);
```
Comment (stanriders) [inline PerformanceCalculator/Profile/ProfileCommand.cs:90]: You're creating a second `PerformanceCalculator` right after the first one (line 87), don't think that's needed

File: PerformanceCalculator/Profile/ProfileCommand.cs [hunk 2]
```diff
@@ -87,12 +87,32 @@ public override void Execute()
...
+
+                var pp = 0.0;
+                var maxCombo = 0.0;
+                var categories = new Dictionary<string, double>();
+
+                try
```
Comment (stanriders) [inline PerformanceCalculator/Profile/ProfileCommand.cs:96]: `try-catch` block here isn't needed

File: PerformanceCalculator/Profile/ProfileCommand.cs [hunk 3]
```diff
@@ -87,12 +87,32 @@ public override void Execute()
...
+                {
+                    pp = perfCalc.Calculate(categories);
```
Comment (stanriders) [inline PerformanceCalculator/Profile/ProfileCommand.cs:98]: This value isn't being used (I think you mean to put it on line 110)

**Review by stanriders — CHANGES_REQUESTED** [review]

**Comment by Finadoggie** [comment]
```
dotnet : An error occurred when executing task 'InspectCode'.
At C:\projects\osu-tools\InspectCode.ps1:26 char:1
+ dotnet cake ./build/InspectCode.cake $cakeArguments
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (An error occurr... 'InspectCode'.:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
Error: 
One or more errors occurred. (inspectcode failed with return code 5)
	inspectcode failed with return code 5
Command executed with exception: One or more errors occurred. (inspectcode failed with return code 5)
	inspectcode failed with return code 5

```
I'm not sure what this error means and/or how to fix it, and the tools work fine when I'm running the commands so idk why it's even firing errors

**Comment by bdach** [comment]
Please see the "messages" tab in the appveyor CI result. There are some static analysis inspections therein to fix.

**Comment by Finadoggie** [comment]
Bdach and StanR, you guys are lifesavers!

**Comment by tybug** [comment]
For future reference, "update profilecommand.cs" isn't a very useful commit message. Better to say something like "fix inspection errors", if in fact that's what that commit is doing.

**Comment by Finadoggie** [comment]
aight, that was just the name it gave all the commits that I didn't name, and I thought it would be assumed that every unnamed update was just a "fix the broken" commit

**Review by smoogipoo — CHANGES_REQUESTED** [review]

**Review by smoogipoo — APPROVED** [review]
lgtm

**Pull request merged**